modules/service-fips.sh

```diff
@@ -50,6 +54,63 @@ fips_disable() {
     not_supported 'Disabling FIPS'
 }
 
+fips_updates_enable() {
+    local token="$1"
+    local bypass_prompt="$2"
+    local fips_configured=0
+    local fips_kernel_version=0
+
+    check_token "$FIPS_UPDATES_REPO_URL" "$token"
+
+    echo "Updating FIPS packages will take the system out of FIPS compliance."
+    if [ "$bypass_prompt" -ne 1 ]; then
+        if ! prompt_user 'Do you want to proceed?'; then
+            error_msg "Aborting updating FIPS packages..."
+            return
+        fi
+    fi
+
+    # add the fips-updates repo if the system is undergoing updates the first time
+    if [ ! -f "$FIPS_UPDATES_REPO_LIST" ]; then
+        apt_add_repo "$FIPS_UPDATES_REPO_LIST" "$FIPS_UPDATES_REPO_URL" "$token" \
+                 "${KEYRINGS_DIR}/${FIPS_UPDATES_REPO_KEY_FILE}"
+        apt_add_repo_pinning "$FIPS_UPDATES_REPO_PREFERENCES" \
+                         LP-PPA-ubuntu-advantage-fips-updates 1001
+        apt_install_package_if_missing_file "$APT_METHOD_HTTPS" apt-transport-https
+        apt_install_package_if_missing_file "$CA_CERTIFICATES" ca-certificates
+        echo -n 'Running apt-get update... '
+        check_result apt_get update
+        echo 'Ubuntu FIPS-UPDATES PPA repository enabled.'
+    fi
+
+    # if a fips package is found on the system, assume fips was configured before
+    # users could be running with fips=0 or fips=1, so just checking package here
+    if apt_is_package_installed fips-initramfs; then
+       fips_configured=1
+       # get the fips kernel version installed here
+       fips_kernel_version=$(package_version linux-fips)
+    fi
+
+    # update all the fips packages
+    echo -n 'Updating FIPS packages (this may take a while)... '
+    # shellcheck disable=SC2086
+    check_result apt_get install $FIPS_HMAC_PACKAGES $FIPS_OTHER_PACKAGES
+
+    # if fips was never configured before and is enabled for the
+    # first time, configure fips
+    if [ "$fips_configured" -eq 0 ]; then
+        echo "Configuring FIPS... "
+        _fips_configure
+    fi
+    echo -n "Successfully updated FIPS packages."
```

Comment:
I think this `-n` here is incorrect. Look at the last line, how my shell prompt is showing up:
```
ubuntu@xenial-ua:~$ sudo ua enable-fips-updates  <redacted> 
Checking token... OK
Updating FIPS packages will take the system out of FIPS compliance.
Do you want to proceed? [N/y] y
Updating FIPS packages (this may take a while)... OK
Successfully updated FIPS packages.ubuntu@xenial-ua:~$ 
```